src/test/regress/expected/multi_insert_select.out

```diff
@@ -142,7 +142,7 @@ FROM
   raw_events_first
 WHERE
   user_id < 0;
-NOTICE:  evaluating on master
+ERROR:  Subqueries without relations are not allowed in INSERT ... SELECT queries
```

Comment:
I think it should be fine. The test is not very meaningful and RTEs without relations may lead to errors. This seems a bugfix to me 😄